### ISSUE TITLE
Class Instrumentation breaks when local variable set to null

### ISSUE THREAD

**Issue by paulseawa** [electronicarts/ea-async#4]
If you initialize a local variable to null, then call an await() within a try catch block, you get ClassNotFoundException on the class in which you did that, during that classes initialization.

Simply by removing the null initialization works around the issue.

```java
  @Async
  public Task<Void> doStuff(Task<Void> otherStuff) {

    String x = null;        //bad
    //String x;             //ok

    try {
      await(otherStuff);
    } catch (Exception e) {
      throw e;
    }
    return Task.done();

  }
```

**Comment by JoeHegarty** [comment]
Thanks for the report. @DanielSperry is going to take a look at this shortly.

**Comment by DanielSperry** [comment]
I'm on it now.

**Comment by DanielSperry** [comment]
fixed. the problem would also occur with:

```
            String x = null;
            await(task);
```

and

```
            String x = null;
            someMethod(x, await(task));
```

and 

```
            someMethod(null, await(task));
```

**Comment by JoeHegarty** [comment]
This is resolved in the 1.0.2 release - https://github.com/electronicarts/ea-async/releases/tag/v1.0.2
